src/web/mjs/connectors/Batoto.mjs

```diff
@@ -18,6 +17,26 @@ export default class Batoto extends AnyACG {
         this.queryMangaLink = 'a.item-title';
         this.queryMangaFlag = 'span.item-flag';
         this.queryChapters = 'div.episode-list div.main a.visited';
+
+        this.config = {
+            url: {
+                label: 'URL',
+                description: `This website's main domain doesn't always work, but has alternate domains.\nThis is the default URL which can also be manually set by the user.`,
+                input: 'text',
+                value: 'https://bato.to'
+            }
+        };
+    }
+
+    get url() {
+        return this.config.url.value;
+    }
+
+    set url(value) {
```

Comment:
Setter seems not to be used anywhere, maybe removed (a user defined value shouldn't be overriden anyway)

Reply:
removed setter